library/src/main/java/eu/kanade/tachiyomi/source/Source.kt

```diff
@@ -21,6 +23,36 @@ interface Source {
      */
     val name: String
 
+    /**
+     * Represents an IETF BCP 47 compliant language tag.
```

Comment:
do we  _need_ to specify regions and scripts in languages?

Reply:
This is just correctly representing what app expected for extensions to return and extensions were returning for `lang` property

Reply:
Do we need the API to be unambiguous? No, but it sounds like good idea.

Reply:
I think @AwkwardPeak7 was under the impression that instead of `en` sources will need to specify `en-us`

Reply:
Considering the comment beneath mentions "all" as a legal tag, shouldn't that be one of the options together with MULTI and OTHER? Or at the very least mentioned in the same bulleted list?

Reply:
Antsy forgot to remove the comment when he removed ALL

Reply:
updated the javadoc

Reply:
test